packages/next/src/client/link.tsx

```diff
@@ -70,12 +70,20 @@ type InternalLinkProps = {
   passHref?: boolean
   /**
    * Prefetch the page in the background.
-   * Any `<Link />` that is in the viewport (initially or through scroll) will be preloaded.
-   * Prefetch can be disabled by passing `prefetch={false}`. When `prefetch` is set to `false`, prefetching will still occur on hover in pages router but not in app router. Pages using [Static Generation](/docs/basic-features/data-fetching/get-static-props.md) will preload `JSON` files with the data for faster page transitions. Prefetching is only enabled in production.
+   * Any `<Link />` that is in the viewport (initially or through scroll) will be prefetched.
+   * Prefetch can be disabled by passing `prefetch={false}`. Prefetching is only enabled in production.
    *
-   * @defaultValue `true`
+   * In App Router:
+   * - `null` (default): For statically generated pages, this will prefetch the full React Server Component data. For dynamic pages, this will prefetch up to the nearest route segment with a [`loading.js`](/docs/app/api-reference/file-conventions/loading) file. If there is no loading file, it will not fetch the full tree to avoid fetching too much data.
```

Comment:
Curious: do these need an absolute URL docs link when it's in JSDoc (e.g., `https://nextjs.org/docs/...`)?

Reply:
Yeah I think they do -- I was copying what we did in other spots (ie [here](https://github.com/vercel/next.js/blob/6dfa1c56241422bf08bd40b035863fd668070884/packages/next/src/client/link.tsx#L42)) but I don't think that's correct so I'll adjust both